pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -253,17 +253,15 @@ func newRuntimeVersion(version string) (*utilversion.Version, error) {
 func (m *kubeGenericRuntimeManager) getTypedVersion() (*runtimeapi.VersionResponse, error) {
 	typedVersion, err := m.runtimeService.Version(kubeRuntimeAPIVersion)
 	if err != nil {
-		klog.Errorf("Get remote runtime typed version failed: %v", err)
```

Comment:
Why removing the log message?

Reply:
To avoid the double-logging issue, the error was both logged and returned